src/common/options.cc

```diff
@@ -3141,7 +3159,7 @@ std::vector<Option> global_options = {
   Option("bluestore_allocator", Option::TYPE_STR, Option::LEVEL_DEV)
   .set_default("bitmap")
   .add_tag("mkfs")
-  .set_description(""),
+  .set_description("Allocator to use for bluestore.  Choices are: bitmap (preferred) and stupid (only for testing)"),
```

Comment:
@markhpc 
I think instead you shall do:

`set_enum_allowed({"bitamp", "stupid"})`

here..